php/meterpreter/meterpreter.php

```diff
@@ -512,8 +512,10 @@ function core_enumextcmd($req, &$pkt) {
 
     global $id2f;
 
-    $id_start = packet_get_tlv($req, TLV_TYPE_UINT)['value'];
-    $id_end = packet_get_tlv($req, TLV_TYPE_LENGTH)['value'] + id_start;
+    $id_start_array = packet_get_tlv($req, TLV_TYPE_UINT);
+    $id_start = $id_start_array['value'];
+    $id_end_array = packet_get_tlv($req, TLV_TYPE_LENGTH);
+    $id_end = $id_end_array['value'] + $id_start;
```

Comment:
This missing `$` was defo picked up in a commit that seems to be lost in the rebase. Sorry! Thank you for fixing that.

Clearly I'm hopeless and have managed to get bitten by the "earlier PHP doesn't index rvalues" issue.